prettier/main_test.ts

```diff
@@ -233,3 +233,267 @@ test(async function testPrettierPrintToStdout(): Promise<void> {
 
   emptyDir(tempDir);
 });
+
+test(async function testPrettierReadFromStdinFormatTS(): Promise<void> {
+  const inputCode = `console.log("abc"  )`;
+  const p1 = Deno.run({
+    args: ["deno", "./prettier/testdata/echox.ts", `${inputCode}`],
```

Comment:
I think we should use Deno.execPath for looking up deno executable. That makes it easy to change deno executable for testing, and also it reduces the trouble in windows.